lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -99,20 +98,33 @@ <h3 class="course-title">
         <div class="course-info">
           <span class="info-university">${course_overview.display_org_with_default} - </span>
           <span class="info-course-id">${course_overview.display_number_with_default}</span>
-          <span class="info-date-block" data-tooltip="Hi">
-          <% time_zone = get_user_time_zone(user) %>
-          % if course_overview.has_ended():
-            ${_("Ended - {end_date}").format(end_date=course_overview.end_datetime_text("SHORT_DATE", time_zone))}
-          % elif course_overview.has_started():
-            ${_("Started - {start_date}").format(start_date=course_overview.start_datetime_text("SHORT_DATE", time_zone))}
-          % elif course_overview.start_date_is_still_default: # Course start date TBD
-            ${_("Coming Soon")}
-          % elif course_overview.starts_within(days=5):   # hasn't started yet
-            ${_("Starts - {start_date}").format(start_date=course_overview.start_datetime_text("DAY_AND_TIME", time_zone))}
-          % else:   # hasn't started yet
-            ${_("Starts - {start_date}").format(start_date=course_overview.start_datetime_text("SHORT_DATE", time_zone))}
-          % endif
-          </span>
+          <%
+            if course_overview.start_date_is_still_default:
+                container_string = _("Coming Soon")
+                date = None
+                time_zone = None
+                format = None
+            else:
+                format = 'shortDate'
+                time_zone = user.preferences.model.get_value(user, "time_zone")
```

Comment:
This is going to make an extra SQL call per course on the user's dashboard.  This won't scale and should be fixed.